sdk/webpubsub/Microsoft.Azure.Functions.Worker.Extensions.WebPubSub/src/Models/Output/SendToAllAction.cs

```diff
@@ -15,7 +15,7 @@ public sealed class SendToAllAction : WebPubSubAction
         /// <summary>
         /// Message to broadcast.
         /// </summary>
-        [JsonConverter(typeof(BinaryDataJsonConverter))]
+        [JsonConverter(typeof(System.BinaryDataJsonConverter))]
```

Comment:
This conflicts with https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.binarydatajsonconverter?view=net-9.0-pp so just explicitly sticking with the one defined in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Microsoft.Azure.WebPubSub.Common/src/Internal/BinaryDataJsonConverter.cs as per offline discussion